packages/frontend-2/components/invite/dialog/project/Row.vue

```diff
@@ -233,25 +237,43 @@ const userId = computed({
   }
 })
 
+const showDropdown = computed(() => {
+  if (props.canInviteNewMembers) {
+    return filteredSuggestions.value.length > 0 && isMenuOpen.value
+  }
+  return (
+    (filteredSuggestions.value.length > 0 || isSearchLoading.value || search.value) &&
+    isMenuOpen.value
+  )
+})
+
 const handleInput = (value: string) => {
   search.value = value
+  if (props.isInWorkspace && props.canInviteNewMembers) {
+    emit('update:modelValue', {
+      ...props.modelValue,
+      email: value
+    })
+  }
 }
 
 const handleClear = () => {
   input.value = ''
   search.value = ''
   selectedUser.value = null
+  emit('update:modelValue', {
+    ...props.modelValue,
+    userId: undefined
+  })
 }
 
 const listboxOptionsStyle = computed(() => {
   const style: CSSProperties = {}
-
   const top = listboxButtonBounding.top.value
   const left = listboxButtonBounding.left.value
   const width = listboxButtonBounding.width.value
-  const height = listboxButtonBounding.height.value
 
-  style.top = `${top + height}px`
+  style.top = `${top + (props.showLabel ? 60 : 32)}px`
```

Comment:
This can be improved by being a better dynamic calculation, but this should do the job for now